src/ert/_c_wrappers/enkf/config/gen_kw_config.py

```diff
@@ -71,53 +32,57 @@ def __init__(
         if not os.path.isfile(parameter_file):
             raise IOError(f"No such file:{parameter_file}")
 
-        c_ptr = self._alloc_empty(key, tag_fmt)
```

Comment:
We have no tests for the IOError's here and perhaps they should be `ConfigValidationError`?

Reply:
Might as well make it collect the errors with ErrorInfo ala the pattern used in `ErtConfig`.

Reply:
Done.